mlflow/utils/databricks_utils.py

```diff
@@ -416,3 +416,59 @@ def get_databricks_host_creds(server_uri=None):
     elif config.token:
         return MlflowHostCreds(config.host, token=config.token, ignore_tls_verification=insecure)
     _fail_malformed_databricks_auth(profile)
+
+
+@_use_repl_context_if_available("mlflowGitRepoUrl")
+def get_git_repo_url():
+    try:
+        return _get_command_context().mlflowGitRepoUrl().get()
```

Comment:
Does `mlflowGitRepoUrl` return anything specific to MLflow, or is it just the URL of a Databricks Repo? If the latter, I think we should drop `mlflow` from the naming (I see you added that recently). Ditto for the other methods.

Reply:
It's to prevent confusion with the URL set for Jobs x Repos integration in the `CommandContext`: https://docs.google.com/document/d/1gjGDu_kJVSeO1SEmyN6x5laQy1yHYYG3eEpbVv-Kx_g/edit#